docs/OV_Runtime_UG/stateful_models_intro.md

```diff
@@ -0,0 +1,90 @@
+   If the original Kaldi model contains RNN-like operations with `stateful` option, then after ModelOptimizer conversion,
+   the resulting OpenVINO model will also contain states, represented by pairs ReadValue/Assign operations.
+
+* [Apply LowLatency2 transformation.](./ways_to_get_stateful_model.md#)
+   If after conversion from original model to OpenVINO representation, the resulting model contains TensorIterator or Loop operations,
+   the LowLatency2 transformation can be applied to get model with states.
+   Note: there are some [specific limitations]() to use the transformation.
+
+* [Apply MakeStateful transformation.](./ways_to_get_stateful_model.md)
+   If after conversion from original model to OpenVINO representation, the resulting model contains Parameter and Result operations,
+   which pairwise have the same shape and element type, the MakeStateful transformation can be applied to get model with states.
+
+* [Create the model via OpenVINO API.](./ways_to_get_stateful_model.md)
+   For testing purposes or for some specific cases, when the ways to get OpenVINO model with states described above are not enough for your purposes,
+   you can use OpenVINO API and create `ov::opset8::ReadValue` and `ov::opset8::Assign` operations directly.
+
+## OpenVINO State API
+
+ Inference Engine has the `ov::InferRequest::query_state` method  to get the list of states from a network and `ov::VariableState` class to operate with states. 
+ Below you can find brief description of methods and the workable example of how to use this interface.
+ 
+ * `std::string get_name() const`
+   returns name(variable_id) of according Variable
+ * `void reset()`
+   reset state to default value
+ * `void set_state(const Tensor& state)`
+   set new value for state
+ * `Tensor get_state() const`
+   returns current value of state
+
+## Example of Stateful Network Inference
+
+The example below demonstrates inference of three independent sequences of data. State should be reset between these sequences.
+
+One infer request and one thread will be used in this example. Using several threads is possible if you have several independent sequences. Then each sequence can be processed in its own infer request. Inference of one sequence in several infer requests is not recommended. In one infer request state will be saved automatically between inferences, but 
+if the first step is done in one infer request and the second in another, state should be set in new infer request manually (using `ov::VariableState::set_state` method).
+
+@sphinxdirective
+
+.. tab:: C++
+
+      .. doxygensnippet:: docs/snippets/ov_network_state_intro.cpp
+         :language: cpp
+         :fragment: [ov:state_api_usage]
+
+@endsphinxdirective
+
+You can find more powerful examples demonstrating how to work with networks with states in speech sample and demo. 
+Descriptions can be found in [Samples Overview](./Samples_Overview.md)
```

Comment:
can we link directly to speech sample? other samples does not show work with states